src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.data.neo4j.username=neo4j
+spring.data.neo4j.password=123
```

Comment:
Password shouldn't be posted online. This should be read from the environmental variable.
`spring.data.neo4j.password=${NEO4J_PASSWORD}`
If you run your app with intelijj you can then set enviroment variables in the Run/Debug configruations window. 
There should be probalby also a property with neo4j host

Reply:
done